app/jobs/webhook_job.rb

```diff
@@ -0,0 +1,17 @@
+class WebhookJob < ApplicationJob
+  queue_as :default
+
+  def perform(check_or_batch, callback_uri)
+    response = Faraday.post do |req|
+      req.url callback_uri
+      req.headers["Content-Type"] = "application/json"
+      req.body = check_or_batch.to_h.to_json
+    end
+
+    if response.status >= 500
```

Comment:
can we get an exception back from faraday when a request fails? it feels like it'd be better to use the [ActiveJob retry mechanism](http://api.rubyonrails.org/classes/ActiveJob/Enqueuing.html) - also worth checking that ActiveJob retry eventually aborts so we don't get into an eternal loop (which we'd fall into with this implementation if a server consistently 500'd)

And finally we probably want to be retrying on more cases than 500